node/utils/allowListAccounts.ts

```diff
@@ -0,0 +1,21 @@
+const appName = process.env.VTEX_APP_ID as string
+
+interface StoreGraphQLSettings {
+  trustedAccounts?: string[]
+}
+
+export const allowedAccounts = async (
```

Comment:
```suggestion
export const allowedAccountsList = async (
```
What do you think about changing the file name to allowedAccountsList too?

Reply:
I'm also wondering if it wouldn't be better to use only one of those expressions: allowed x trusted.
I've noticed that some places are mentioning _allowed_ and others mention _trusted_, but I believe they have the same meaning everywhere.

Reply:
What do you think about `trustedAccountsList`?

Reply:
Since it's a predicate, i changed it to `isTrustedAccount`, does it make sense for you two?!

Reply:
Yes, totally!